apps/dialtone-documentation/docs/.vuepress/baseComponents/tokens/TokenTable.vue

```diff
@@ -68,27 +69,14 @@
           <div class="d-body--sm">
             {{ description }}
           </div>
+          <token-value
+            v-if="!showValueColumn()"
```

Comment:
why not using directly `isSmallDevice`?

```suggestion
            v-if="!isSmallDevice"
```